sway-lsp/src/core/session.rs

```diff
@@ -86,6 +86,12 @@ pub struct Session {
     pub diagnostics: Arc<RwLock<Diagnostics>>,
 }
 
+impl Default for Session {
```

Comment:
Do we need both `default()` and `new()`?

Reply:
I only added this to appease clippy. Seems like this lint has recently been added.